src/osd/OSD.cc

```diff
@@ -4353,6 +4353,10 @@ int OSD::shutdown()
     cct->_conf.apply_changes(nullptr);
   }
 
+  // stop MgrClient earlier as it's more like an internal consumer of OSD
+  // should occur before unmounting the database in fast-shutdown to avoid a race condition
```

Comment:
nit: could add the tracker reference here as well

Reply:
Fixed!